roundabout/piece.js

```diff
@@ -72,11 +71,9 @@ export async function findEquivalentCarCids (piece, fetchClaims = createClaimsCl
       continue
     }
     // an equivalence claim may have the pieceCid as the content cid _or_ the equals cid
-    // so check both properties for the car cid.
-    const carCid = asCarCid(claim.equals) ?? asCarCid(claim.content)
-    if (carCid) {
-      cids.add(carCid)
-    }
+    // so if content does not equal piece, we can grab the content. Otherwise equals
+    const equivalentCid = !claim.content.equals(piece) ? claim.content : claim.equals
```

Comment:
IDK, maybe a little easier to read?:

```suggestion
    const equivalentCid = claim.content.equals(piece) ? claim.equals : claim.content
```